### ISSUE TITLE
Renamed assembly is still called CSScriptLibrary.v3.5

### ISSUE THREAD

**Issue by maettu-this** [oleg-shilo/cs-script#29]
The assembly \lib\Bin\NET 3.5\CSScriptLibrary.dll which got renamed from CSScriptLibrary.v3.5.dll unfortunately is still called "CSScriptLibrary.v3.5", e.g. see CSScriptLibrary.xml:
```
    <assembly>
        <name>CSScriptLibrary.v3.5</name>
    </assembly>
```
As a consequence, compiling leads to "Assembly "CSScriptLibrary.v3.5, Version=3.20.0.0, Culture=neutral,...could not be loaded". Reverting the file name back to CSScriptLibrary.v3.5.dll works again.

Please fix the assembly's name, then build and release again, thanks!
(It's not urgent for me, I can live with reverting the file's name.)

**Comment by oleg-shilo** [comment]
Thank you. Yes the `assembly name` vs. ` assembly file name` has gotten wrong in this case. Actually I also discovered that the whole `\lib\Bin\NET 3.5` has this problem (both executables).

The build batch file is corrected and the next release will be fine.

**Comment by oleg-shilo** [comment]
Fixed in Release v3.21.1
